src/js/tabs/trust.controller.js

```diff
@@ -406,6 +406,8 @@ TrustTab.prototype.angular = function (module)
       {
         $scope.trust.loading = true;
         $scope.load_notification('remove_gateway');
+        $scope.removing = true;
```

Comment:
```
$scope.removing = true;
$scope.saving = false;
```
Is there any case where it's saving and removing at the same time? Probably not, in which case it's better to have `$scope.state` thing which can get `saving`, or `removing` values.